File: 0001-Two-Sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> indices;
        for (int i = 0; i < nums.size(); i++) {
            bool found = false;
            for (int j = 0; j < nums.size(); j++) {
                if ((nums[i] + nums[j] == target) && (i != j)) {
                    indices = {i , j};
                    found = true;
                    break;
                }
            }
            if (found) {break;}
        }
        return indices;
    }
};

File: Java/0001-Two_Sum.java
import java.util.HashMap;

class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> values = new HashMap<>();
        int[] indices = new int[2];

        for (int i = 0; i < nums.length; i++) {
            if (values.get(target - nums[i]) == null) {
                values.put(nums[i], i);
            }
            else {
                indices[0] = i;
                indices[1] = values.get(target - nums[i]);
                break;
            }
        }
        
        return indices;
    }   
}